config/network.go

```diff
@@ -120,6 +158,14 @@ func (n *NetworkConfig) UpperCaseNetworkNames() {
 
 	for i, network := range n.SelectedNetworks {
 		n.SelectedNetworks[i] = strings.ToUpper(network)
```

Comment:
this will mess up things further down the lane... my idea was to ignore the case as I find it unnecessarily limiting to have to stick to upper or lower case, can we keep that approach?

Reply:
I did not change this.

Reply:
who did? :D